src/js/custom.js

```diff
@@ -246,3 +246,44 @@ if (typeof NProgress != 'undefined') {
         NProgress.done();
     });
 }
+
+// Language DataTable and Add Collumn 
+var addCollumnTable;
+$(document).ready(function(){
+  	addCollumnTable = $("#datatable-responsive-collumn").dataTable( {           
+        "aoColumnDefs": [
+            { "bVisible": false, "aTargets": [ 7, 8 ] }
+        ],
+        "aaSorting": [[1, 'asc']],
+            "aLengthMenu": [
+            [5, 15, 20, -1],
+            [5, 15, 20, "Todos"] // change per page values here
+        ],
+        // set the initial value
+        "iDisplayLength": 10,
+        "oLanguage": {
+                "sProcessing": '<i class="fa fa-coffee"></i>&nbsp;Aguarde...',
+                "sLengthMenu": "_MENU_ Quantidade",
+                "sInfo": "Total de _TOTAL_ registros - de _START_ até _END_",
```

Comment:
I think this shouldn't be include as DataTable has a full translation of these option on multiple languages on json format here: https://datatables.net/plug-ins/i18n/